### DISCUSSION TITLE
Use custom mail template per submitter

### DISCUSSION THREAD

**Discussion by waza-ari** [docusealco/docuseal#310] [Q&A] [answered]
We're evaluating docuseal for our non profit organization, where two parties need to sign a document (the person which received a security training and the trainer). First of all, thanks so much for the work and an outstanding piece of software!

Ideally, we'd like to be able to customise the mail template that is used to send the PDF to the first party (person received the training) and to the trainer differently. Looking at the API, this doesn't seem to be possible right now.

Did I miss something, is that possible somehow?

Actually, I see this might be a PRO feature, is that correct?

<img width="336" alt="image" src="https://github.com/docusealco/docuseal/assets/4508976/e35cd026-8a83-4ec4-b15a-baa21c15e8c8">

**Comment by omohokcoj** [comment, marked as answer]
@waza-ari thanks for your question - it's possible to customize email message for each party by passing the message: { body:, subject: } param in the submitters: [] list on the POST /submissions API:
https://www.docuseal.co/docs/api#create-a-submission (it's not documented but the message: param works for each submitter)

**Reply by waza-ari** [reply]
Thanks! Works perfectly! For reference, if anyone else is looking for it:

```json
{
    "template_id": 1,
    "order": "preserved",
    "submitters": [
        {
            "name": "Submitter 1",
            "role": "Trainee",
            "email": "mail-address-1",
            "values": {},
            "message": {
                "subject": "Custom Subject",
                "body": "Hi there,\n\nnewlines are escaped regularly. Use the following link to sign:\n\n{{submitter.link}}\n\nBest regards"
            }
        },
        {
            "role": "Trainer",
            "email": "<mail 2>",
        }
    ]
}
```
